protos/dcs/unit/unit.proto

```diff
@@ -20,6 +20,11 @@ service UnitService {
 
   // https://wiki.hoggitworld.com/view/DCS_func_getByName
   rpc Get(GetRequest) returns (GetResponse) {}
+
+  /**
+   * Get information about the unit in 3D space, including its position, orientation and velocity.
+   */
+  rpc Export(ExportRequest) returns (ExportResponse) {}
```

Comment:
The naming is still up for discussion. I don't think that the API is too specialized anymore to my LSO use-case as it contains all the data necessary to export any unit to e.g. a ACMI (Tacview) file.

Other naming idea: `GetTransform`

Reply:
Yeah, I am not sure on the `Export` name, it is very... generic. `GetTransform` seems to be the name for this kind of stuff in graphics and physics libraries so shall we go with that?